crypten/__init__.py

```diff
@@ -236,7 +236,9 @@ def load(f, preloaded=None, encrypted=False, dummy_model=None, src=0, **kwargs):
 
         # source party
         if comm.get().get_rank() == src:
-            result = preloaded if preloaded else torch.load(f, **kwargs)
+            assert not (f is None and preloaded is None), "Load failed: f or preloaded should be specified"
```

Comment:
If someone adds ```preloaded``` then we should not pass a loading function

Reply:
This seems reasonable. 

We're currently working on refactoring the `load` function to deprecate `preloaded` and separate it into a new function.

Reply:
In this case, should those changes be merged or should we wait for the refactoring?